### DISCUSSION TITLE
Administration Page is not mobile friendly at all

### DISCUSSION THREAD

**Discussion by TheExkaliburg** [requarks/wiki#6528] [Error / Bug Report] [answered]
I've just noticed that if i visit the administration page on mobile, most of my screen is just filled with the navbar from the left. There is no intuitive way to close this navbar and make room for the settings.

![image](https://github.com/requarks/wiki/assets/34279353/7626f768-5be2-4fa0-9bc0-f22c4859edc3)

Otherwise great job, mobile has some more minor problems at a lot of different places (f.e. editing pages), but this was the worst i found till now.

**Comment by NGPixel** [comment, marked as answer]
The admin page is desktop only. This will be revisited in v3 to make it mobile-friendly.
